diff -Naur A/core/arch/arm/plat-imx/conf.mk B/core/arch/arm/plat-imx/conf.mk
--- A/core/arch/arm/plat-imx/conf.mk	2020-08-05 17:19:54.253349049 +0530
+++ B/core/arch/arm/plat-imx/conf.mk	2020-08-05 17:21:41.045696983 +0530
@@ -17,7 +17,7 @@
 mx8m-flavorlist = mx8mqevk
 mx8mm-flavorlist = mx8mmevk
 mx8mn-flavorlist = mx8mnevk
-mx8qm-flavorlist = mx8qmmek mx8qmlpddr4arm2
+mx8qm-flavorlist = mx8qmmek mx8qmlpddr4arm2 mx8qm_iwg27m
 mx8qx-flavorlist = mx8qxpmek mx8qxplpddr4arm2
 
 ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6ul-flavorlist)))
@@ -451,6 +451,11 @@
 CFG_DDR_SIZE ?= 0x80000000
 CFG_UART_BASE ?= UART0_BASE
 endif
+
+ifneq (,$(filter $(PLATFORM_FLAVOR),mx8qm_iwg27m))
+CFG_DDR_SIZE ?= 0x80000000
+CFG_UART_BASE ?= UART0_BASE
+endif
 
 ifneq (,$(filter $(PLATFORM_FLAVOR),mx8qmlpddr4arm2))
 CFG_DDR_SIZE ?= 0x80000000
